dataset/data.go

```diff
@@ -496,7 +503,12 @@ func (m Metadata) ToString() string {
 	if m.RelatedDatasets != nil {
 		b.WriteString(fmt.Sprintf("Related Links: %s\n", *m.RelatedDatasets))
 	}
-
+	if m.CanonicalTopic != nil{
+		b.WriteString(fmt.Sprintf("Canonical Topic: %s\n", *m.CanonicalTopic))
+	}
+	if m.SubTopics != nil {
```

Comment:
Should we check the length of the array here too? ie. if empty, don't display it? 

Does the `Subtopics` need to be a pointer anyway?

Reply:
Not really, just following the pattern, but I've changed this now :+1: